### PR TITLE
Implemented i18n

### LINKED ISSUE #23: Translate the app
- The app should be made available in German and English (maybe other languages you speak?)
- The user must be able to switch the language in the GUI

Start by reading the docs at https://docs.djangoproject.com/en/2.0/topics/i18n/, and the three links in that document (translation of text, formatting of dates, times and numbers, and time zones (even if we don't need that yet. Time zones will be another issue).

### PR DESCRIPTION
**Pull request by stevenpi**
Implemented i18n and tranlated all the pages into english and german since this was part of the issue  
Added a `settings` page to change the language of the webpage.  

Resolves #23 

New Settings Page:

settingspage

(German) Translated homepage:

tranlatedhome

### CODE REVIEW

File: locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: Blog/blogForms.py:14
+msgid "Max 64 characters"
+msgstr ""
```
Comment (mbaechtold) [inline locale/de/LC_MESSAGES/django.po]: Could this be translated too?

File: locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+
+#: Blog/blogForms.py:15
+msgid "Max 2048 characters"
+msgstr ""
```
Comment (mbaechtold) [inline locale/de/LC_MESSAGES/django.po]: Could this be translated too?

File: locale/de/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+
+#: Blog/blogForms.py:19
+msgid "Title"
+msgstr ""
```
Comment (mbaechtold) [inline locale/de/LC_MESSAGES/django.po]: Could this be translated too?

File: locale/de/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+
+#: Blog/blogForms.py:20
+msgid "Content"
+msgstr ""
```
Comment (mbaechtold) [inline locale/de/LC_MESSAGES/django.po]: Could this be translated too?

File: locale/de/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+
+#: Blog/templates/Blog/comment_add.html:5
+msgid "Add Comment"
+msgstr ""
```
Comment (mbaechtold) [inline locale/de/LC_MESSAGES/django.po]: Could this be translated too?

**Comment by stevenpi** [comment]
Added the `.mo` file and finished the `.po` file as requested

**Pull request merged**